data/json/items/resources/stone.json

```diff
@@ -29,6 +29,20 @@
     "name": { "str": "flint" },
     "description": "A chunk of flint, the kind of rock that can make sparks if stuck against high-carbon steel.  It also chips in a semi-predictable way if struck, leaving a sharp edge."
   },
+  {
+    "type": "GENERIC",
+    "id": "field_stone",
+    "symbol": ";",
+    "color": "brown",
+    "name": { "str": "field stone" },
+    "description": "A medium, flatish, heavy rock, big enough and flat enough to be used as a into field stone walls.",
```

Comment:
seems like youre missing a bit in the end of the description. used as a what?

Reply:
Thanks! I’ll need to rewrite that